lib/module.js

```diff
@@ -8,6 +8,12 @@ const defaults = {
 module.exports = function module(moduleOptions) {
   const options = Object.assign({}, defaults, this.options.trailingslash, moduleOptions);
 
+  if (!this.options.router) {
+    this.options.router = {}
+  }
+
+  this.options.router.trailingSlash = true // https://nuxtjs.org/api/configuration-router/#trailingslash
```

Comment:
@manniL @WilliamDASILVA 
Why do you need to force trailing slash? In my project, I don't set `this.options.trailingslash`(it's `undefined` by default). So, I expected to get a redirect from the page with '/' to the page without '/'. And this PR brokes the main logic(all pages instead of main starts to return 404 after these changes).

With enabled `this.options.router.trailingSlash` you enforce trailing slash for all your pages, but we want our pages to work without trailing slash.

Reply:
Yeah, this module is broken for me now. If we redirect to non-trailing slash URLs but tell the router to expect trailing slashes, it only gives 404 pages. I've pinned this dependency to `1.1.0` until this is changed.